test/sagemaker_tests/pytorch/training/conftest.py

```diff
@@ -564,6 +564,19 @@ def skip_p5_tests(instance_type, processor, ecr_image):
             pytest.skip("P5 EC2 instance require CUDA 12.0 or higher.")
 
 
+@pytest.fixture(autouse=True)
+def skip_smdataparallel_p5_tests(instance_type, processor, ecr_image):
```

Comment:
You'll need to change the arguments to `skip_smdataparallel_p5_tests(request, processor, ecr_image,  efa_instance_type)`. These args will be pulled from the tests parameters and inputs [here](https://github.com/aws/deep-learning-containers/pull/3730/files#diff-16b42d6ad46ca750063ba41a3e3bca3c19f3a4fc07ad87d80d9022c5654592dfL51)